portfolio/src/main/webapp/messages.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Mia Chen</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
+</head>
+
+<body onload="showMessages()">
```

Comment:
I am not sure if you linked this paged somewhere in the index or the other pages. Maybe have a link to this page on the contact.html page etc?